cli/src/lib/install.ts

```diff
@@ -24,32 +24,46 @@ export default async function Update(pkg: string) {
   let plugins: string[] = pkgJSONContents?.grouparoo?.plugins;
 
   if (!plugins) {
-    logger.fail("There is no `grouparoo` section in this package.json.");
+    logger.fail(
+      "There is no `grouparoo.plugins` section in this package.json."
+    );
     process.exit(1);
   }
 
-  if (pkg && pkg?.match("@grouparoo/ui-")) {
-    const existingUIPackage = plugins.find((p) => p.match(/@grouparoo\/ui.*/));
-    if (existingUIPackage) {
-      logger.fail(
-        "There is already a ui package in this project. Uninstall the existing ui package before adding another."
-      );
-      process.exit(1);
-    }
+  const installedPackages = Object.keys(
+    pkgJSONContents?.devDependencies || []
```

Comment:
What does Object.keys() of an array produce?

Reply:
I don't think this is right... if we are running on `NODE_ENV=production`, and we haven't installed devDependencies, we shouldn't include `pkgJSONContents.devDependencies` right?

Reply:
But... this is just for `grouparoo install`, so maybe it's OK

Reply:
> What does Object.keys() of an array produce?

`dependencies` and `devDependencies` come through as objects where the key is the plugin name and the value is the version identifier. `Object.keys()` here gives us a list of names.

> I don't think this is right... if we are running on `NODE_ENV=production`, and we haven't installed devDependencies, we shouldn't include `pkgJSONContents.devDependencies` right?

I think what I was trying to protect against was still have you get an error message when trying to installed both production ui packages. But because those are the prod packages, I think you're right — I don't think there's any reason to look in `devDependencies`.

> But... this is just for `grouparoo install`, so maybe it's OK

Yeah, in the end, this is just to check against installing both ui production packages, so it probably doesn't matter, but it could definitely be tighter. I'll clean it up.